src/Raven.Server/Rachis/Leader.cs

```diff
@@ -84,6 +85,7 @@ public Leader(RachisConsensus engine, long term)
             Term = term;
             _engine = engine;
             PeersVersion[engine.Tag] = ClusterCommandsVersionManager.MyCommandsVersion;
+            _threadName = $"Consensus Leader - {_engine.Tag} in term {Term}";
```

Comment:
Shouldn't we use a short thread name?

Reply:
In TrySetThreadPriority it is used only to log in case of a failure, short thread name is not needed